src/db/tables/subscribers.js

```diff
@@ -510,6 +510,33 @@ async function deleteOnerepProfileId (subscriberId) {
 }
 /* c8 ignore stop */
 
+/* c8 ignore start */
+/**
+ * @param {string} fxaId
+ */
+// Not covered by tests; mostly side-effects. See test-coverage.md#mock-heavy
+/* c8 ignore start */
+async function incrementSignInCount (fxaId) {
+  return await knex('subscribers')
+    .where('fxa_uid', fxaId)
+    .increment("sign_in_count", 1)
+}
```

Comment:
I’m wondering if it might be enough to only keep track of the count for users with a OneRep profile. As we only need to rely on the count for Plus (eligible) users after their initial sign-up we could also only start counting from then onwards. Or does this make things unnecessarily complicated?

Reply:
Everyone who runs a free scan also has a OneRep profile no? To determine plus status, we'd actually need to fetch that from the fxa subscription array? 

Ah I thought we'd display this for everyone. Let me take a look at the requirements again

Reply:
Yes, users who run a free scan also have a OneRep profile. With “Plus (eligible)” users I mean free users that are in the US and can potentially upgrade to Plus. In order to make implementing the current requirements easier I think we only care about the login count after a user runs a scan. Theoretically, users could authenticate multiple times before reaching the dashboard if they don’t run the scan and I think we could ignore those?